src/Paket.Core/Installation/RestoreProcess.fs

```diff
@@ -462,7 +463,8 @@ let Restore(dependenciesFileName,projectFile,force,group,referencesFileNames,ign
                     ProjectFile.FindAllProjects root
                     |> Seq.filter (fun proj -> proj.GetToolsVersion() >= 15.0)
                     |> Seq.iter (fun proj ->
-                        RestoreNewSdkProject dependenciesFile lockFile resolved groups proj
+                        let objDir = objDir |> Option.defaultValue (Path.Combine(Path.GetDirectoryName(proj.FileName),"obj"))
```

Comment:
I assume this path is called if I just do "paket restore"? Then we would need to crack the project file and manually parse "$(BaseIntermediateOutputPath)". :\